ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Mock/UTxO.hs

```diff
@@ -46,14 +50,21 @@ import           Ouroboros.Consensus.Ledger.Mock.Address
   Basic definitions
 -------------------------------------------------------------------------------}
 
-data Tx = Tx (Set TxIn) [TxOut]
-  deriving (Show, Eq, Ord, Generic, Serialise)
+data Tx = UnsafeTx (Set TxIn) [TxOut]
```

Comment:
Oh, can't we use a pattern synonym for this? I don't like having to pattern-match on an `UnsafeTx` :grin: 
So keep this constructor, don't export it, but add a bidirectional `Tx` pattern synonym that replaces `mkTx` and is the same as `UnsafeTx` pattern-wise.

Reply:
Done.